public/css/imprint.css

```diff
@@ -0,0 +1,59 @@
+/* imprint.css */
+
+.imprint {
+  position: relative;
+  max-width: 800px;
+  margin: 0 auto;
+  padding: 20px;
+  font-family: Arial, sans-serif;
+  line-height: 1.5;
+  color: #333;
+}
+
+.navbar {
+  position: absolute;
+  top: 0;
+  left: 0;
```

Comment:
When opening dev tools for example, the logo overlaps the text. Is this intended?